src/app/modules/payments/__tests__/stripe.service.spec.ts

```diff
@@ -689,7 +692,11 @@ describe('stripe.service', () => {
     })
   })
   describe('linkStripeAccountToForm', () => {
-    it('should call func to attach payment account information', async () => {
+    beforeEach(() => {
+      jest.restoreAllMocks()
+    })
+
+    it('should attach payment account information for new accounts', async () => {
```

Comment:
Add a test case for when the feature flag has not been enabled - stripe account should still be linked successfully.

Reply:
It's covered in https://github.com/opengovsg/FormSG/pull/6604/files#diff-2925cc24c9a1ab9f17fa6f5ee6f8873de02f8f9bc2900e5d01ff9f1c8210566cR823

Reply:
(just noticed a typo on the name)